app/models/manageiq/providers/storage_manager/cinder_manager/refresh_helper_methods.rb

```diff
@@ -0,0 +1,173 @@
+module ManageIQ::Providers::StorageManager::CinderManager::RefreshHelperMethods
```

Comment:
Why did you move these into a separate file? These aren't "helper methods" these methods are doing the actual parsing.